style.css

```diff
@@ -1,27 +1,23 @@
 #navbar {
-    position: fixed;
```

Comment:
- [ ] Kindly organize CSS spaces because it would make your code look more professional and cleaner.